src/components/ec-metroline-item/ec-metroline-item.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-metroline-item` : 'ec-metroline-item'"
+    class="ec-metroline-item"
+    :class="{
+      'ec-metroline-item--active': isActive || ( isCompleted && isLast),
+      'ec-metroline-item--is-last': isLast
+    }"
+  >
+    <div
+      data-test="ec-metroline-item__status"
+      class="ec-metroline-item__status"
+    >
+      <div
+        data-test="ec-metroline-item__status-disk"
```

Comment:
Since it contains an icon as well maybe is tiny bit better to go for plain ```badge``` will swap to that so will become ```ec-metroline-item__badge```.